docker/bixarena/services/apex.yml

```diff
@@ -14,3 +14,5 @@ services:
     depends_on:
       bixarena-app:
         condition: service_started
+      bixarena-postgres:
+        condition: service_healthy
```

Comment:
This should be a dependency of the app, and later on possibly of the upcoming api service. The DB is not a direct dependency of apex.